test/SymbolGraph/Symbols/Implicit.swift

```diff
@@ -0,0 +1,8 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift %s -module-name Implicit -emit-module -emit-module-path %t/
+// RUN: %target-swift-symbolgraph-extract -module-name Implicit -I %t -pretty-print -output-dir %t -minimum-access-level internal
+// RUN: %FileCheck %s --input-file %t/Implicit.symbols.json
+
+public class SomeClass {}
+
+// CHECK: "precise": "s:8Implicit9SomeClassCACycfc"
```

Comment:
Not sure I follow how this tests removing the `isImplicit` check. Is this the USR of a synthesised member?

Reply:
Also, should we have a test that verifies that when `-minimum-access-level public` is set, implicit members don't show up?

Reply:
> Also, should we have a test that verifies that when `-minimum-access-level public` is set, implicit members don't show up?

Implicit initializers aren't necessarily tied to `-minimum-access-level` - this one happens to be marked `internal` as part of its definition. I can add a line to the test that ensures that it doesn't appear when the access level is `public`, but that's not necessarily the spirit of the change.

Reply:
I've updated the commit with a comment, a test for when the access level is `public`, and an additional test for implicit struct initializers.